fairroot/online/steer/FairRunOnline.h

```diff
@@ -55,8 +55,6 @@ class FairRunOnline : public FairRun
      *  is not checked anymore after initialization
      */
 
-    /** Init containers executed on PROOF, which is part of Init when running locally*/
-    void InitContainers();
     void SetContainerStatic(Bool_t tempBool = kTRUE);
     Bool_t GetContainerStatic() { return fStatic; };
```

Comment:
Can you take a look at `SetContainerStatic`/`GetContainerStatic` as well? They look unused as well?

As InitContainers (and the *Static ones) are public, could you consider adding a (breaking) notice to the ChangeLog?

Reply:
@karabowi In R3B scenario one always had to update parameters when RunID was changing.

But for generic use case, I agree with you, that we should check for `! fStatic`.

Reply:
I have discussed the issue with both @kresan and @fuhlig1 , and we agreed that `fStatic` should be used in a proper way in `FairRunOnline`. Appropriate commit will be attached to this PR later today.